onnx/defs/tensor/utils.cc

```diff
@@ -98,8 +109,11 @@ void resizeShapeInference(InferenceContext& ctx) {
   const TensorProto* scales = 2 < ctx.getNumInputs() ? ctx.getInputData(2) : nullptr;
   const TensorProto* sizes = 3 < ctx.getNumInputs() ? ctx.getInputData(3) : nullptr;
 
+  // specific to opset11 where scales input is not optional but can be a empty tensor
```

Comment:
May be we should add an extra `int opset` parameter to the method, so that we can customize the behavior for each opset as needed.

Reply:
Is the empty tensor input supported only for opset11? Or, also later?

Reply:
empty string is only supported since opset13. with opset11, 'scales' is not optional and thus cannot use empty string.
According to text description of scales in opset13+, I think empty tensor cannot be used to express "no scales input", only empty sting can be used for the purpose. 
Given that, I will use 'int opset' parameter for the method.